js/ui/windowManager.js

```diff
@@ -1451,7 +1481,7 @@ const WindowManager = new Lang.Class({
          * any artifacts from shadows and the like */
         Tweener.addTween(dst, {
                     opacity: 255,
-                    time: WINDOW_ANIMATION_TIME,
+                    time: WINDOW_ANIMATION_TIME * 4,
```

Comment:
To me this works better with the fading in of the window - a smoother transition.